lib/CRMS/RightsPredictor.pm

```diff
@@ -69,11 +70,13 @@ sub new {
 sub last_source_copyright {
   my $self              = shift;
   my $death_or_pub_date = shift || ''; # Reviewer-supplied
+  my $is_pub            = shift;
```

Comment:
Please put a short comment to contextualize `$is_pub`, like the rest of the inputs.